lib/streamlit/hello/pages/2_🌍_Mapping_Demo.py

```diff
@@ -83,7 +83,6 @@ def from_data_file(filename):
         if selected_layers:
             st.pydeck_chart(
                 pdk.Deck(
-                    map_style="mapbox://styles/mapbox/light-v9",
```

Comment:
I think we need to bring back this until we get a `theme` parameter. Otherwise, it will use the pydeck default dark theme instead.

Reply:
Good catch! FIxed on 7799f2e